.travis.yml

```diff
@@ -2,7 +2,7 @@ language: go
 
 go:
   - 1.6
-  - tip
+  - 1.7
```

Comment:
Basically tip gets unstable after a release - probably half way through 1.8
we'll turn tip on.  Otherwise would be failing all the time.

On Tue, Aug 16, 2016 at 12:41 PM, Steve Kuznetsov <notifications@github.com>
wrote:

> In .travis.yml
> <https://github.com/openshift/origin/pull/10458#discussion_r74972318>:
>
> > @@ -2,7 +2,7 @@ language: go
> >
> >  go:
> >    - 1.6
> > -  - tip
> > +  - 1.7
>
> I thought you argued to keep tip?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/openshift/origin/pull/10458/files/4b046250753fe0e53bf15c100edb397d81ac6315#r74972318>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABG_pwAOcjhcjIDk_08kGQixr2ueJzNVks5qgehHgaJpZM4Jlmyi>
> .
>